shared_model/bindings/model_query_builder_embind.cpp

```diff
@@ -11,56 +11,58 @@ using namespace shared_model::proto;
 
 using namespace shared_model::crypto;
 
-EMSCRIPTEN_BINDINGS(model_query_builder)
-{
+EMSCRIPTEN_BINDINGS(model_query_builder) {
   register_vector<Hash>("HashVector");
-  
-  class_<Hash, base<Blob>>("Hash")
-  .constructor<const std::string&>()
-  .function("toString", &Hash::toString)
-  ;
 
+  class_<Hash, base<Blob>>("Hash").constructor<const std::string &>().function(
+      "toString", &Hash::toString);
+
+  // TODO 20.06.18 stinger112: Add underlaying types for commented methods
```

Comment:
jira id is missing

Reply:
It's not a preferred or planned task now. 
We don't know where we will do it. 
So, before I deleted this comments with todos, then return it after your review. It can not be even implemented at all if we decide so.

What should I do?